packages/php/blueprint/src/Importers/ImportInstallTheme.php

```diff
@@ -81,17 +87,25 @@ public function process( $schema ): StepProcessorResult {
 			$this->result->add_error( "Failed to install theme '$theme->slug'." );
 		}
 
-		$theme_switch = true === $theme->activate && $this->wp_switch_theme( $theme->slug );
-
-		if ( $theme_switch ) {
-			$this->result->add_info( "Switched theme to '$theme->slug'." );
-		} else {
-			$this->result->add_error( "Failed to switch theme to '$theme->slug'." );
-		}
+		$this->activate_theme( $schema );
 
 		return $this->result;
 	}
 
+	protected function activate_theme( $schema ) {
+		$theme = $schema->themeZipFile;
```

Comment:
Not related to this PR, but I think we should use snake case format for variables.

Reply:
@chihsuan this is actually Blueprint spec. We don't have control over it.